src/scene.rs

```diff
@@ -166,6 +188,15 @@ impl<I: ImageSize> Scene<I> {
         total
     }
 
+    /// Get the number of running animations for a sprite. If sprite does not
+    /// exist, returns 0.
```

Comment:
Dont unwrap it, match against it for the failure case.

There are 2 different meanings of '0' in this method, one to mean that the sprite has 0 running animations, and another to mean that the sprite was not found.

Rust has 0 cost enums (through `NonZero<T>` to make things like this more explicit.

'not wanting to unwrap it' does not excuse laziness, and I feel that the correct behaviour should be maintained here.